apps/examples/testcase/le_tc/kernel/tc_pthread.c

```diff
@@ -1539,6 +1540,58 @@ static void tc_pthread_pthread_setcanceltype(void)
 }
 #endif
 
+/**
+* @fn                   :tc_pthread_pthread_mutex_consistent
+* @brief                :This tc tests pthread_mutex_consistent()
+* @Scenario             :the pthread_mutex_consistent() function can be used to mark the state protected by the mutex referenced by mutex as consistent again
+*                        and this situtaion arises, If an owner of a robust mutex terminates while holding the mutex,
+*                        the mutex becomes inconsistent and the next thread that acquires the mutex lock shall be notified of the state.
+*                        In this case, the mutex does not become normally usable again until the state is marked consistent.
+* @API'scovered         :pthread_setcanceltype
+* @Preconditions        :none
+* @Postconditions       :none
+* @return               :void
+*/
+static void *test_mutex_consistent(void *param)
+{
+	int ret = -1;
+	ret = pthread_mutex_lock(&g_mutex);
+	if (ret != 0) {
+		printf("pthread_mutexlock was not successful, errno = %d", ret);
+	}
+}
```

Comment:
The return type of this function is void pointer so that you need to add `return NULL;` at the end of the function.